ml/dune-project

```diff
@@ -0,0 +1,3 @@
+(lang dune 3.0)
+
+(cram enable)
```

Comment:
I don't think this is mandatory anymore as of dune 3.0

Reply:
correct, TIL